spec/lib/landable/bunny_messaging_service.rb

```diff
@@ -0,0 +1,8 @@
+class BunnyMessagingService
+  def initialize
```

Comment:
I agree with @mwesigwa. This should either move to the dummy app (if you need it for the test), or I dont think we should need this.  I think we should just have documentation that you should have a publish method when you configure whichever AMPQ you use.